dashboard/app/models/user.rb

```diff
@@ -568,6 +566,23 @@ def username_required?
     provider == User::PROVIDER_MANUAL
   end
 
+  def update_without_password(params, *options)
+    if params[:races]
+      update_columns(races: params[:races].join(','))
+      if params[:races].include? 'closed_dialog'
+        update_columns(races: 'closed_dialog')
+      end
+      if params[:races].length > 5
```

Comment:
Presumably the nonsense-filter can or will be removed from where it was previously?